tests/core/test_solve.py

```diff
@@ -48,7 +48,7 @@ def get_solver(tmpdir, specs_to_add=(), specs_to_remove=(), prefix_records=(), h
     spec_map = {spec.name: spec for spec in history_specs}
     get_index_r_1(context.subdir)
     with patch.object(History, 'get_requested_specs_map', return_value=spec_map):
-        solver = Solver(tmpdir, (Channel('channel-1'),), (context.subdir,),
+        solver = _get_solver_logic()(tmpdir, (Channel('channel-1'),), (context.subdir,),
```

Comment:
Can we make this a fixture? It would also make it easier to parameterize the test.

Reply:
Yes, agreed!